java-agent/src/nativeTest/kotlin/com/epam/drill/agent/configuration/ValidatorsTest.kt

```diff
@@ -0,0 +1,64 @@
+package com.epam.drill.agent.configuration
+
+import com.epam.drill.konform.validation.Invalid
+import com.epam.drill.konform.validation.Valid
+import com.epam.drill.konform.validation.Validation
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class ValidatorsTest {
+    private val hostAndPort = Validation<SomeObject> {
+        SomeObject::str required {
+            hostAndPort()
+        }
+    }
+    private val identifier = Validation<SomeObject> {
+        SomeObject::str required {
+            identifier()
+        }
+    }
+    private val validPackage = Validation<SomeObject> {
+        SomeObject::str required {
+            isValidPackage()
+        }
+    }
+
+
+    @Test
+    fun `check host and port matcher`() {
```

Comment:
Change test name to reflect case. (Applies to all tests in the file).
Example:
`fun 'check doStuff to work'` - unclear
`fun 'given input X doStuff must succeed'`,  `fun 'given input X doStuff must fail'` - clear